feature/urpf/otg_tests/urpf_vrf_test/urpf_vrf_test.go

```diff
@@ -0,0 +1,559 @@
+
+	// ATE interfaces
+	atePort1 = attrs.Attributes{Name: "ateP1", IPv4: "192.0.2.2", IPv6: "2001:db8:1::2", MAC: "02:00:02:01:01:01", IPv4Len: plenIPv4, IPv6Len: plenIPv6}
+	atePort2 = attrs.Attributes{Name: "ateP2", IPv4: "192.0.2.6", IPv6: "2001:db8:2::2", MAC: "02:00:04:01:01:01", IPv4Len: plenIPv4, IPv6Len: plenIPv6}
+
+	// Prefixes advertised by ATE
+	// Valid source prefixes advertised from ATE Port 1
+	ateAdvIPv4Prefix1 = "198.18.1.0"
+	ateAdvIPv6Prefix1 = "2001:db8:10::"
+	prefix1Len        = 24
+	prefix1LenV6      = 64
+
+	// Invalid source prefixes advertised from ATE Port 1 (but rejected by DUT policy)
+	ateAdvIPv4Prefix2 = "198.18.2.0"
+	ateAdvIPv6Prefix2 = "2001:db8:20::"
+	prefix2Len        = 24
+	prefix2LenV6      = 64
+
+	// Destination prefixes advertised from ATE Port 2
+	ateAdvIPv4Prefix3 = "198.18.3.0"
+	ateAdvIPv6Prefix3 = "2001:db8:30::"
+	prefix3Len        = 24
+	prefix3LenV6      = 64
+
+	dstAddr       = []string{GUEDstIPv4}
+	defaultNIName = strings.ToLower("DEFAULT")
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// configureDUT configures the DUT with interfaces, a non-default VRF, BGP, route policies for leaking and rejecting routes, and uRPF.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) *gnmi.SetBatch {
+	t.Helper()
+	p1 := dut.Port(t, "port1")
+	p2 := dut.Port(t, "port2")
+	intBatch := new(gnmi.SetBatch)
+	t.Logf("Configuring Interfaces")
+	configureDUTInterface(t, dut, intBatch, &dutPort1, p1, true)
+	configureDUTInterface(t, dut, intBatch, &dutPort2, p2, false)
+
+	configureDUTLoopback(t, dut, intBatch)
+	t.Log("Configuring Hardware Init")
+	configureHardwareInit(t, dut)
+	// configureGueEncap(t, dut)
+
+	cfgplugins.EnableDefaultNetworkInstanceBgp(t, dut, dutAS)
+	fptest.ConfigureDefaultNetworkInstance(t, dut)
+
+	t.Log("Configuring Network Instances")
+	defaultNI := cfgplugins.ConfigureNetworkInstance(t, dut, defaultNIName, isDefaultVRF)
+	nonDefaultNI := cfgplugins.ConfigureNetworkInstance(t, dut, nonDefaultVRF, !isDefaultVRF)
+	cfgplugins.ConfigureBGPNeighbor(t, dut, nonDefaultNI, dutPort1.IPv4, atePort1.IPv4, dutAS, ateAS1, "IPv4", true)
+	cfgplugins.ConfigureBGPNeighbor(t, dut, nonDefaultNI, dutPort1.IPv4, atePort1.IPv6, dutAS, ateAS1, "IPv6", true)
+	cfgplugins.ConfigureBGPNeighbor(t, dut, defaultNI, dutPort2.IPv4, atePort2.IPv4, dutAS, ateAS2, "IPv4", true)
+	cfgplugins.ConfigureBGPNeighbor(t, dut, defaultNI, dutPort2.IPv4, atePort2.IPv6, dutAS, ateAS2, "IPv6", true)
+
+	cfgplugins.UpdateNetworkInstanceOnDut(t, dut, defaultNIName, defaultNI)
+	cfgplugins.UpdateNetworkInstanceOnDut(t, dut, nonDefaultVRF, nonDefaultNI)
+	configureDUTPort(t, dut, intBatch, &dutPort1, p1, nonDefaultVRF)
+	intBatch.Set(t, dut)
+	return intBatch
+}
+
+// configureRouteLeaking configure/delete route leaking.
+func configureRouteLeaking(t *testing.T, dut *ondatra.DUTDevice, enable bool) {
+	t.Helper()
+	if deviations.NetworkInstanceImportExportPolicyOCUnsupported(dut) {
```

Comment:
Move the configuration into a cfgplugin function named after the OC named feature such as `NewInterInstancePolicy`

Then inside `NewInterInstancePolicy` place the deviation which can generate the CLI instead of the OC.

See reference https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/README.md